src/mig/modules/pkgprint/templates.go

```diff
@@ -0,0 +1,91 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Aaron Meihm ameihm@mozilla.com [:alm]
+package pkgprint
+
+import (
+	"fmt"
+	"regexp"
+)
+
+type ppTemplate struct {
```

Comment:
If the end goal is to make these loadable from the command line, you may want to export all fields in the `ppTemplate` and `fingerprint` structs.